File: app/src/main/java/ms/imagine/foodiemate/api/AzurePrediction.java
package ms.imagine.foodiemate.api;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class AzurePrediction {
    @SerializedName("Id")
    @Expose
    public String id;

    @SerializedName("Project")
    @Expose
    public String project;

    @SerializedName("Iteration")
    @Expose
    public String iteration;

    @SerializedName("Created")
    @Expose
    public String created;

    @SerializedName("Predictions")
    @Expose
    public List<Prediction> predictions;

    @Override public String toString() {
        return id + project + iteration + created;
    }
}

File: app/src/main/java/ms/imagine/foodiemate/api/AzurePrediction.kt
package ms.imagine.foodiemate.api

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class AzurePrediction {
    @SerializedName("Id")
    @Expose
    var id: String? = null
    @SerializedName("Project")
    @Expose
    var project: String? = null
    @SerializedName("Iteration")
    @Expose
    var iteration: String? = null
    @SerializedName("Created")
    @Expose
    var created: String? = null
    @SerializedName("Predictions")
    @Expose
    var predictions: List<Prediction>? = null

    override fun toString(): String {
        return id + project + iteration + created
    }
}
